scripts/publish_doxygen_coverage.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+const jwt = require('jsonwebtoken');
+const github = require('@octokit/rest')();
+const zlib = require('zlib');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+
+const SIZE_CHECK_APP_ID = 14028;
+const SIZE_CHECK_APP_INSTALLATION_ID = 229425;
+
+const coverage = JSON.parse(fs.readFileSync(process.argv[2]));
```

Comment:
Error handling? `JSON.parse()` can fail if the input is not valid JSON. Also, make sure that given file `i.e process.argv[2]` is not missing.